enterprise/backend/src/metabase_enterprise/scim/v2/api.clj

```diff
@@ -0,0 +1,14 @@
+(ns metabase-enterprise.scim.v2.api
+  "/api/ee/scim/v2/ endpoints. These are the endpoints which implement the SCIM protocol, as opposed to SCIM
+  configuration endpoints which are in `metabase-enterprise.scim.api`."
```

Comment:
Wasn't sure about the namespace names and API routes here. The actual SCIM protocol is conventionally at `/scim/v2` but we also have normal APIs for creating/deleting keys, which I've put at `/scim`. Can change this approach if anyone has a better idea.